Tests/XCUITests/PocketTests.swift

```diff
@@ -10,13 +10,10 @@ class PocketTest: BaseTestCase {
         waitForExistence(app.staticTexts[AccessibilityIdentifiers.FirefoxHomepage.SectionTitles.pocket])
         XCTAssertEqual(app.staticTexts[AccessibilityIdentifiers.FirefoxHomepage.SectionTitles.pocket].label, "Thought-Provoking Stories")
 
-        // There should be two stories on iPhone and three on iPad
+        // There should be at least 8 stories on iPhone and three on iPad
```

Comment:
Nit: we may need to update also the three on iPad... since the assertion is for stories >7, but this can be done in another PR no need to re-run for this